server/src-lib/Hasura/RQL/DDL/Schema/Cache.hs

```diff
@@ -406,9 +405,6 @@ buildSchemaCacheRule = proc (catalogMetadata, invalidationKeys) -> do
 -- if not, incorporates them into the schema cache.
 withMetadataCheck :: (MonadTx m, CacheRWM m, HasSQLGenCtx m) => Bool -> m a -> m a
 withMetadataCheck cascade action = do
-  -- Drop hdb_views so no interference is caused to the sql query
-  liftTx $ Q.catchE defaultTxErrorHandler clearHdbViews
```

Comment:
We should still keep this because Event Trigger trigger functions are still created in `hdb_views`. The proposal to move them to `hdb_catalog` is pending discussion.

Reply:
Oops, thanks. Updated ..